.github/workflows/deploy_branch.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   build:
     name: Build staging branch
-    uses: zooniverse/Panoptes-Front-End/.github/workflows/npm_build.yml@production-release
+    uses: zooniverse/Panoptes-Front-End/.github/workflows/npm_build.yml@clean-up-npm-caching
```

Comment:
I've changed this to test the changes here. It can change back to `production-release` before this is merged.

Reply:
```suggestion
    uses: zooniverse/Panoptes-Front-End/.github/workflows/npm_build.yml@production-release
```